packages/env/lib/config.js

```diff
@@ -128,7 +128,12 @@ module.exports = {
 				themes: [],
 				port: 8888,
 				testsPort: 8889,
-				config: { WP_DEBUG: true, SCRIPT_DEBUG: true },
+				config: {
+					WP_DEBUG: true,
+					SCRIPT_DEBUG: true,
+					WP_TESTS_DOMAIN: 'example.org',
+					WP_PHP_BINARY: 'php',
```

Comment:
I'm not sure this is working :thinking: -- at least not for me :slightly_frowning_face: 

This is the `readConfig` function, which is used by `init-config.js` to write the `docker-compose.yml` file (via `build-docker-compose-config.js`). But AFAICS, none of those files evaluate the contents of the `config`  object. And if I check my `~/wp-env/12345678/docker-compose.yml`, I don't see any of those config constants either. :thinking:

Reply:
Right, the constants are not added to the docker compose config, but set with wp-cli:
https://github.com/WordPress/gutenberg/blob/066d8390989c40fe2f6e8fd79adc4c1546a28165/packages/env/lib/wordpress.js#L95-L112

This function is called during `wp-env start`